src/components/dashboard/Claim.js

```diff
@@ -35,6 +42,12 @@ type ClaimState = {
 
 const log = logger.child({ from: 'Claim' })
 
+// eslint-disable-next-line require-await
+const openLearnMoreLink = async () => openLink(Config.learnMoreEconomyUrl)
```

Comment:
i've refactored the similar open link handler:
https://github.com/GoodDollar/GoodDAPP/commit/ec13f76e20fb711581ae7eef1aced606b5298e48#diff-6318ed8af39dbb5bd54ab5802c94357bR36
pls do the same way

Reply:
done